app/views/partials/foyer/capture-montant-ressource.html

```diff
@@ -1,35 +1,44 @@
-<div class="panel panel-montant {{ ressourceType.category }}">
+<div class="panel ressource-input {{ ressourceType.category }}">
   <div class="panel-heading">
     <h3 class="panel-title">{{ ressourceType.label }}</h3>
   </div>
   <div class="panel-body">
-    <div class="row ressources-values">
-      <div class="col-sm-3" ng-repeat="month in months">
-        <label for="{{ ressource.type.id }}-month_{{ $index + 1 }}">
+    <div class="months"
+      ng-class="{ 'has-error': form.submitted && form[ressource.type.id + '-year'].$invalid }"
```

Comment:
This should be `$min` rather than `$invalid`, otherwise any kind of error on `year` leads to all fields being highlighted.

![capture d ecran 2017-06-27 a 16 08 26](https://user-images.githubusercontent.com/222463/27591736-ea73f8da-5b52-11e7-889d-088410c3cace.png)